base/Dockerfile

```diff
@@ -1,9 +1,13 @@
 FROM debian:sid
 MAINTAINER Zero Cho "http://itsze.ro"
 
-RUN apt-get update
-RUN apt-get install -y scala git curl openjdk-7-jdk netcat
+RUN apt-get update && apt-get install -y \
+  curl \
+  openjdk-8-jre-headless \
```

Comment:
Sounds good. All we'd need to add would be netcat. Wanna raise a PR? (I'm
traveling today)